pkgs/top-level/python-packages.nix

```diff
@@ -12273,6 +12289,8 @@ self: super: with self; {
 
   pyric = callPackage ../development/python-modules/pyric { };
 
+  pyright = callPackage ../development/python-modules/pyright { };
```

Comment:
Adding pyright to python packages seems to be causing many packages to rebuild.
It looks like crewai-tools only depends on it for development and is not imported anywhere. I think you could remove pyright and remove the dependency from crewai-tools using something like this:

```nix
pythonRemoveDeps = [
  "pyright"
];
```

Additional resource: https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonrelaxdepshook-using-pythonrelaxdepshook

Reply:
Would you remove `pkgs/development/python-modules/pyright/default.nix` too?